README.md

```diff
@@ -51,6 +51,12 @@ Here's a rough list of Adobe XD views of the main routes:
 
 ---
 
+## Contributing
+
+- Please visit our [Contributing Documenation](https://github.com/QueerGlobal/qg-frontend-v2/blob/main/CONTRIBUTING.md) on how you can help out! ty!
```

Comment:
Should we add the [qg-docs](https://github.com/QueerGlobal/qg-docs) repo, or would that complicate things?